FileSystem/Workbook.js

```diff
@@ -24,3 +41,73 @@ function openWorkbook(path) {
   openURL(file.getUrl(), file.getName() + ' file open in new tab');
   return file;
 }
+
+/**
+ * Prompts the user to provide the local file path for the current active spreadsheet.
+ * @param {boolean} error Error flag indicating whether an error occurred
+ */
+function promptActiveWorkbookPath(error) {
+  // Get current active spreadsheet details
+  var driveId = SpreadsheetApp.getActive().getId();
+  var path = getFullDrivePath(driveId, true);
+  var pathSplit = path.split('/');
+  pathSplit.pop(); // Remove file name from path
+  var drivePath = pathSplit.join('/');
+  pathSplit.shift(); // Remove "Drive" or "My Drive" from path
+  // Generate possible local path
+  var localPathExample = 'C:\\Documents\\' + pathSplit.join('\\');
```

Comment:
Use constants
C:\\Documents\\'

Reply:
This is the only place I am going to use it, I am just building a helpful message here. It is not really the actual windows file path...

Reply:
Even then it is recommended to use Constants.

Reply:
Okay, got it, Fixed it